modules/gapi/include/opencv2/gapi/garray.hpp

```diff
@@ -368,6 +368,8 @@ template<typename T> class GArray
     detail::GArrayU m_ref;
 };
 
+using GArrayP2f = GArray<cv::Point2f>;
```

Comment:
Can it be defined as shadow?

Reply:
No, that symbol should be in libopencv_gapi.so